### DISCUSSION TITLE
Determine version of SQLCipher used in SQLitePCLRaw.bundle_e_sqlcipher 2.1.4

### DISCUSSION THREAD

**Discussion by MortenBoysen** [ericsink/SQLitePCL.raw#534] [Q&A] [answered]
Hi

I have been trying to find out what version of SQLCipher is used by then newest release (2.1.4) of `SQLitePCLRaw.bundle_e_sqlcipher` (by referincing `SQLitePCLRaw.lib.e_sqlcipher` 2.1.4). I have been trying to find out, but I simple cannot find the information. For SQLite it clearly states the version as 3.40.1 in the release notes. Is this information available for SQLCipher as well?

**Comment by ericsink** [comment, marked as answer]
For the 2.1.4 release, the e_sqlcipher builds are unchanged, remaining at SQLCipher 4.5.2, which is based on SQLite 3.39.2.  AFAICT, SQLCipher has not yet done a release based on 3.40.1.

**Reply by MortenBoysen** [reply]
Thanks for the quick and precise answer. I was wondering, whether it it is possible to have this information as part of the release notes in future releases of `SQLitePCLRaw.bundle_e_sqlcipher`?

**Reply by ericsink** [reply]
My general practice is to make sure the release notes indicate which version is included.  I initially didn't do that for this release because ...

Never mind.  Yes, I'll try to remember to be explicit about this for each release.

**Reply by MortenBoysen** [reply]
Thanks. I really appreciate it.

And thank you for all the work you put into maintaining this. It is much appreciated.
